src/main/resources/sql/V1__moderation.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS moderation
```

Comment:
Maybe we can add timestamp, what did you think ?

Reply:
yeah sure